### ISSUE TITLE
add support for flex properties

### ISSUE THREAD

**Issue by Delapouite** [wavded/vim-stylus#46]
Hi

Some recently introduced CSS properties would be a nice additions to this plugin : `flex-grow`, `justify-content`, `align-items`…

**Comment by iansinnott** [comment]
I agree it would be nice if this worked out of the box, but these properties are still getting picked up as `stylusProperty`, which means you can manually link their syntax highlighting style. In your vimrc:

```
hi link stylusProperty cssVisualProp
```

You can also run this in a window and see the result. Worked for me to make flex properties highlight the same as everything else.

**Comment by talmobi** [comment]
@iansinnott works perfectly! Thanks!
